PhysicsTools/PatAlgos/python/slimming/lostTracks_cfi.py

```diff
@@ -10,14 +12,16 @@
     primaryVertices = cms.InputTag("offlineSlimmedPrimaryVertices"),
     originalVertices = cms.InputTag("offlinePrimaryVertices"),
     muons = cms.InputTag("muons"),
-    minPt = cms.double(0.95),	
-    minHits = cms.uint32(8),	
-    minPixelHits = cms.uint32(1),  
-    covarianceVersion = cms.int32(0), #so far: 0 is Phase0, 1 is Phase1   
-    covarianceSchema = cms.int32(0), #old miniaod like
+    minPt = cms.double(0.95),
+    minHits = cms.uint32(8),
+    minPixelHits = cms.uint32(1),
+    covarianceVersion = cms.int32(0), #so far: 0 is Phase0, 1 is Phase1
+    covariancePackingSchemas = packedPFCandidates.covariancePackingSchemas,
     qualsToAutoAccept = cms.vstring("highPurity"),
     minPtToStoreProps = cms.double(0.95),
-    passThroughCut = cms.string("pt>2")                        
+    passThroughCut = cms.string("pt>2"),
+    pvAssignment = primaryVertexAssociation.assignment,
+    useLegacySetup = cms.bool(False) #When True: check only if track used to fit vertex[0] and do not store track detailed info for Pt between 0.5 and minPtToStoreProps GeV
```

Comment:
One should check if this flag does actually reproduce the past behaviour. 
As a first attempt, I'd set it to "true" and run the bot tests: if there are no differences, that will already be a good indicator, and the flag can be reverted to "false" in the master.
Still, since the updates integrated in this PR are non-trivial, before doing any backport I would make a larger stat test with a sensible sample.

Reply:
I did such tests locally with success.
But, to understand better are you going to set the flag to True manually to run tests or I should change it in the PR for the tests and then revert?